app/views/vendors/_showsales.html.erb

```diff
@@ -0,0 +1,11 @@
+<h3><%= title %></h3>
```

Comment:
In general, we want to separate multi-word view names with the underscore, so this one would be `_show_sales`